app/views/contexts/summary_context.py

```diff
@@ -14,11 +14,18 @@ def __call__(
         groups = list(self._build_all_groups(return_to))
         summary_options = self._schema.get_summary_options()
         collapsible = summary_options.get("collapsible", False)
+        headers = (
+            ["Question", "Answer given", "Change answer"]
+            if answers_are_editable
+            else ["Question", "Answer given"]
+        )
```

Comment:
We should mark these strings for translation.

Reply:
(Non blocker) I would avoid the duplication:
```suggestion
        headers = ["Question", "Answer given"]
        if answers_are_editable:
            headers.append("Change answer")
```